web/styles/popovers.css

```diff
@@ -1297,6 +1296,29 @@ ul {
     }
 }
 
+#help-menu-dropdown,
+#gear-menu-dropdown,
+#personal-menu-dropdown {
+    .simplebar-content {
+        min-width: var(--navbar-popover-menu-min-width);
+    }
+}
+
+#message-actions-menu-dropdown {
+    --popover-menu-max-width: 350px;
+
+    .simplebar-content {
+        min-width: var(--message-actions-popover-min-width);
+    }
+}
+
+#stream-actions-menu-popover,
+#topic-actions-menu-popover {
+    .simplebar-content {
+        min-width: var(--stream-topic-popover-min-width);
```

Comment:
Looking at the IDs this is nested under, I wonder if `actions` doesn't also belong in this variable?

Reply:
Yeah I think `topic-actions-popover-min-width` is most correct

Reply:
Hello, sorry I'm a little late on this one. Is it an intentional decision not to include `stream` in this new variable name?